submissions/adesokandijam

```diff
@@ -0,0 +1 @@
+
```

Comment:
Kindly include your submission for week one task in this PR. More on this here https://github.com/CottaCush/Dufuna/blob/master/WEEKLY_TASKS.md